admin-portal/src/App.tsx

```diff
@@ -41,15 +34,32 @@ function App() {
     MedicalNeedsService.http = http;
     PledgeService.http = http;
     SupplierService.http = http;
-  }, [state.isAuthenticated]);
+  }, []);
```

Comment:
Moved the `http` client instantiation to a standalone hook. IMO, we don't need to reinitialize when the user's authenticated state changes.

Please validate.